Makefile

```diff
@@ -3,20 +3,20 @@ test:
 testcov:
 	py.test --cov=ideascube/ --cov-report=term-missing
 develop:
-	pip install -r requirements-dev.txt
```

Comment:
> I did that because in my Dockerfile, I tried not to use a virtualenv.

And `pip` or `python` was still pointing to python2? Was your venv made with python3 and sourced?